### PR TITLE
OCPBUGS-28812: kubelet.service: add delegate to service

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

**Comment by harche** [comment]
/retitle OCPBUGS-28812: kubelet.service: add delegate to service

**Comment by haircommander** [comment]
hmm I understand why this works (it means libcontainer called by kubelet setting the cgroupfs won't have its values clobbered by systemd), but I am dubious as to whether this is the approach we want. From my perspective,  I feel like we shouldn't delegate to kubelet because we're using the systemd cgroup manager, and expect systemd to do the cgroup configuration. Did something change in el8->el9 that meant we formerly were delegating and are no longer? if not, I'm worried this will have knock on effects.

**Comment by harche** [comment]
> hmm I understand why this works (it means libcontainer called by kubelet setting the cgroupfs won't have its values clobbered by systemd), but I am dubious as to whether this is the approach we want. From my perspective, I feel like we shouldn't delegate to kubelet because we're using the systemd cgroup manager, and expect systemd to do the cgroup configuration. Did something change in el8->el9 that meant we formerly were delegating and are no longer? if not, I'm worried this will have knock on effects.

Thanks @haircommander . We need to understand this better. 

/hold

File: templates/master/01-master-kubelet/_base/units/kubelet.service.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ contents: |
 
   [Service]
   Type=notify
+  Delegate=true
```
[the remark this answers is not in the document]
Comment (harche) [reply templates/master/01-master-kubelet/_base/units/kubelet.service.yaml:12]: /cc @rphillips

**Pull request closed without merging**